packages/server/src/ui/visual/HeroBar/index.tsx

```diff
@@ -3,11 +3,17 @@ import InstallIcon from './install-icon.svg';
 // tslint:disable-next-line:no-implicit-dependencies
 import background from '!url-loader!./background-image.svg';
 
-export function InstallButton({size = 'sm'}: {size?: 'sm' | 'lg'}) {
+export function InstallButton({
+  size = 'sm',
+  shadow = 'shadow-gray',
+}: {
+  size?: 'sm' | 'lg';
+  shadow?: 'shadow-gray' | 'shadow-orange' | 'shadow-white';
+}) {
   return (
     <a
       href="https://github.com/apps/rollingversions/installations/new"
-      className={`flex items-center justify-center bg-black text-white italic font-poppins font-black ${
+      className={`flex items-center justify-center bg-black text-white italic font-poppins font-black focus:outline-none focus:${shadow} ${
```

Comment:
The optimiser will not be able to look at `focus:${shadow} and figure out that the `focus:shadow-gray` class is needed.

Reply:
OK